web/prql-codemirror-demo/src/codemirror.ts

```diff
@@ -0,0 +1,113 @@
+import {
```

Comment:
Is this file copy-pasted from somewhere? We should link back to it if so...

Reply:
Yes, it is. This file is from the `basic-setup` repository.
https://github.com/codemirror/basic-setup/blob/main/src/codemirror.ts

We could also use the [`codemirror`](https://www.npmjs.com/package/codemirror) npm package instead, but this lets users pick and chose which extensions to load by modifying their own copy of the `codemirror.ts` file.

Reply:
Let's link back to it, then

Reply:
Great, thank you!